<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>specific_situation</class>
 <widget class="QDialog" name="specific_situation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1652</width>
    <height>975</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCustomPlot" name="plot" native="true">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QDateTimeEdit" name="dateTimeEdit_end"/>
         </item>
         <item row="0" column="1">
          <widget class="QDateTimeEdit" name="dateTimeEdit_start"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>End</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Start    </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dateTimeEdit_start</tabstop>
  <tabstop>dateTimeEdit_end</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>horizontalSlider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
